### PR TITLE
Create wetandpissy.yml

### PR DESCRIPTION
**Pull request by JaseNZC**

### CODE REVIEW

**Review by SpedNSFW — CHANGES_REQUESTED** [review]
File needs to be in the scrapers directory. Also can you please add it to the SCRAPERS-LIST.md file please.

File: wetandpissy.yml [hunk 1]
```diff
@@ -0,0 +1,29 @@
+name: wetandpissy
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - wetandpissy.com
```
Comment (SpedNSFW) [inline wetandpissy.yml:5]:
```suggestion
      - simplyanal.com
      - weliketosuck.com
      - wetandpissy.com
      - wetandpuffy.com
```

File: wetandpissy.yml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        postProcess:
+          - parseDate: Jan 2, 2006
+      Details: //div[@class="movie-description"]/text()[1]
+      Tags:
+        Name: //div[@class='tags-box']//a/text()
+      Performers:
+        Name: //span[@class="model-movie"]//a/text()
+      Studio:
+        Name:
+          fixed: wetandpissy
```
Comment (SpedNSFW) [inline wetandpissy.yml:27]:
```suggestion
          selector: //img[@class="logo-overley"]/@src|//ul[@class="nav-list"]/li/a/img/@src
          postProcess:
            - replace:
              - regex: .+?\.(.+?)\..+
                with: $1
```

File: wetandpissy.yml [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      Title:
+        selector: //div[@class="header-title"]/h2
+        postProcess:
+          - replace:
+            - regex: ^(.*)[—]\s+
+              with:
+      Date:
+        selector: //span[@class="date-movie"]/b
+        postProcess:
+          - parseDate: Jan 2, 2006
```
Comment (SpedNSFW) [inline wetandpissy.yml:19]:
```suggestion
          - parseDate: Jan 02, 2006
```

**Comment by JaseNZC** [comment]
Bugger sorry thought it was in the scrapers dir no idea how to move it now.
SCRAPERS-LIST.md updated and PR issued (guess I was supposed to do that)

**Review by SpedNSFW — CHANGES_REQUESTED** [review]
This scraper is applicable to some of the other sites on the Puffy Network. Make these changes and rename the file to `PuffyNetworkSite.yml`. Also add the new sites to the SCRAPER-LIST.md file.

File: SCRAPERS-LIST.md [hunk 4]
```diff
@@ -755,9 +756,12 @@ watchmygf.me|WatchMyGF.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Tube Site
 watchreal.com|Hustler.yml|:heavy_check_mark:|:x:|:x:|:x:|CDP|-
 webyoung.com|GammaEntertainment.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Lesbian
 wefuckblackgirls.com|DogFart.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
+weliketosuck.com|PuffyNetworkSite.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 welivetogether.com|RealityKingsOL.yml|:heavy_check_mark:|:x:|:x:|:x:|-|Lesbian
 weneednewtalents.com|Teencoreclub.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
 westcoastproductions.com|AdultEmpireCash.yml|:heavy_check_mark:|:x:|:heavy_check_mark:|:x:|-|-
+wetandpuffy.com|PuffyNetworkSite.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
+wetandpissy.com|PuffyNetworkSite.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
```
Comment (SpedNSFW) [inline SCRAPERS-LIST.md:764]:
```suggestion
wetandpissy.com|PuffyNetworkSite.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
wetandpuffy.com|PuffyNetworkSite.yml|:heavy_check_mark:|:x:|:x:|:x:|-|-
```

**Review by SpedNSFW — CHANGES_REQUESTED** [review]

File: scrapers/PuffyNetworkSite.yml [hunk 5]
```diff
@@ -0,0 +1,36 @@
+name: wetandpissy
```
Comment (SpedNSFW) [inline scrapers/PuffyNetworkSite.yml:1]: Should change this to PuffyNetworkSite too now.
```suggestion
name: PuffyNetworkSite
```

**Review by SpedNSFW — CHANGES_REQUESTED** [review]

**Comment by JaseNZC** [comment]
Sorry guys first time and all , got there in the end with a lot of help thank you.

**Review by SpedNSFW — APPROVED** [review]

File: scrapers/PuffyNetworkSite.yml [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+      Title:
+        selector: //div[@class="header-title"]/h2
+        postProcess:
+          - replace:
+            - regex: ^(.*)[—]\s+
+              with:
+      Date:
+        selector: //span[@class="date-movie"]/b
+        postProcess:
+          - parseDate: Jan 02, 2006
```
Comment (bnkai) [inline scrapers/PuffyNetworkSite.yml:22]:
```suggestion
          - parseDate: Jan 2, 2006
```
`https://www.wetandpuffy.com/videos/jane-toy/`
`https://www.wetandpissy.com/videos/video-nurse-naomi/`
The 0 breaks the parsing for date in most scenes. Was there a case where 0 was needed?

**Comment by bnkai** [comment]
Also jsut mentioning that the parent site <www.puffynetwork.com> which we have a scraper for already seems to have the dates even for the sites above that dont use them.

File: scrapers/PuffyNetworkSite.yml [hunk 7]
```diff
@@ -0,0 +1,36 @@
...
+        Name: //div[@class='tags-box']//a/text()
+      Performers:
+        Name: //span[@class="model-movie"]//a/text()
+      Studio:
+        Name:
+          selector: //img[@class="logo-overley"]/@src|//ul[@class="nav-list"]/li/a/img/@src
+          postProcess:
+            - replace:
+              - regex: .+?\.(.+?)\..+
+                with: $1
```
Comment (bnkai) [inline scrapers/PuffyNetworkSite.yml:34]:
```suggestion
          postProcess:
            - replace:
                - regex: .+?\.(.+?)\..+
                  with: $1
            - map:
                weliketosuck: We Like To Suck
                wetandpuffy: Wet And Puffy
                wetandpissy: Wet And Pissy
                simplyanal: Simply Anal
```
The above change keep the studio names consistent with the parent studio scraper

**Pull request merged**